client/me/profile/index.jsx

```diff
@@ -100,7 +100,7 @@ const Profile = createReactClass( {
 
 						<FormFieldset>
 							<FormLabel htmlFor="display_name">
-								{ this.props.translate( 'Public Display Name' ) }
+								{ this.props.translate( 'Public display name' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **60** times:
`translate( 'Public Display Name' )` *ES Score: 9*